
use_icp: true  #If false only uses calculated guess

topic_combined_scan: /merged_cloud
topic_cloud_filtered: /filtered_point_cloud
topic_cloud_environment: /environment_cloud
topic_target_tranformation: /target_transformation
topic_transformed_cloud: /transformed_cloud
topic_table_template_cloud_combined: /table_template_cloud_combined

frame_base_link_id: base_link

two_legs_distance_threshold: 1

filter_mean_k: 2
filter_stdDev_threshold: 1
filter_radius_search: 0.1 
filter_min_neighbours_in_radius:  2 

angle_dev_treshold: 0.01 #circa 0.6Â°
parallel_dev_threshold: 0.005 # 5mm
accepted_distance_treshold: 0.01 # 2cm

max_acceleration_outside: 0.1
max_acceleration_inside: 0.05

min_y_distance_table_legs: 0.005  #0.0025 #0.0075
block_rotation_below: 0.5 #this is necessary when only_drive_forward_below is disabled.
only_drive_forward_below: 0.75 #set to 0 to disable just driving inside

wait_after_lift_cmd: 1 #1 # 5.0  
# In sec. For now doing lift -cmds is implemented with waiting
# Otherwise we would have to read the joint states.
# /evo_robot/joint_states [sensor_msgs/JointState] - probably.
